awesome_object_store/__init__.py

```diff
@@ -19,6 +19,8 @@ def init_object_store(
     if protocol == "gcs":
         return GoogleCloudStore(bucket, logger)
     else:
+        assert isinstance(host, str), ValueError("Host must be provided for MinioStore")
```

Comment:
The check here is to fix lint issue, because input host type can be string or None, but MinioStore accept only string 

I'll change like this
```
if host is None:
    ValueError("Host must be provided for MinioStore")

return MinioStore(
    bucket,
    str(host),
    access_key,
    secret_key,
    secure,
    region,
    logger,
)
```